examples/config/cosmodc2_config.yml

```diff
@@ -26,17 +26,18 @@ TXSourceTrueNumberDensity:
 TXMainMaps:
     chunk_rows:  100000
     pixelization: healpix
-    nside: 512
```

Comment:
Could we leave these values at their previous ones for the default pipeline?  Could add a second config file maybe for the high res and true shear version.

Reply:
actually, can leave these as they are I think, this is the CosmoDC2 file after all which has high number density.

Reply:
I mean, leave your changes in place.